.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Report reproducible software issues so we can improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**System information**
+ - Cantera version: [e.g. 2.4]
+ - OS: [e.g. Windows 10]
+ - Python/MATLAB version:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
In the Markdown format, you need to leave an extra blank line to get this to appear in a new paragraph.

Reply:
Same comment for the other paragraphs, too.